assets/src/blocks/Spreadsheet/SpreadsheetEditor.js

```diff
@@ -0,0 +1,117 @@
+import {useState} from '@wordpress/element';
+import {InspectorControls} from '@wordpress/block-editor';
+import ColorPaletteControl from '../../components/ColorPaletteControl/ColorPaletteControl';
+import {SpreadsheetFrontend} from './SpreadsheetFrontend';
+import {debounce} from 'lodash';
+import {TextControl, PanelBody} from '@wordpress/components';
```

Comment:
Could we move all the `@wordpress` deps at the top of the file, then other dependency, and finally the modules?

Reply:
Yes we could, but is it really important? If it is then we should open a ticket to change that in other blocks too, because we don't respect that everywhere.